genrl/deep/agents/a2c/a2c_temp.py

```diff
@@ -0,0 +1,169 @@
+from typing import Any, Dict, Optional, Tuple, Union
+
+import gym
+import numpy as np
+import torch
+import torch.nn.functional as F
+import torch.optim as opt
+from torch.autograd import Variable
+
+from ....environments.vec_env import VecEnv
+from ...common import OnPolicyAgent, RolloutBuffer, get_env_properties, get_model
+
+
+class A2C(OnPolicyAgent):
+    def __init__(
+        self,
+        network_type: str,
+        env: Union[gym.Env, VecEnv],
```

Comment:
This is same in all agents, so can be extracted out with *args I guess

Reply:
Yea I was thinking of this initially but then these are really important variables, I think should be kept out.